GVFS/GVFS.Platform.Windows/ETWTelemetryEventListener.cs

```diff
@@ -105,22 +109,38 @@ private EventSourceOptions CreateOptions(EventLevel level, Keywords keywords, Ev
             return options;
         }
         
-        // Needed to pass relatedId without metadata
         [EventData]
-        public struct EmptyStruct
+        public struct Payload
         {
+            public Payload(string enlistmentId, string mountId)
+            {
+                this.EnlistmentId = enlistmentId;
+                this.MountId = mountId;
+            }
+
+            [EventField]
+            public string EnlistmentId { get; }

```

Comment:
Have we confirmed that these will be ok if they are null?

Reply:
In my testing these would be serialized as blank if null.